docs/source/test_network.md

```diff
@@ -420,17 +419,15 @@ org2.example.com
 + set +x
 ```
 
-However, `network.sh` also provides the option to bring up the network using
+However, the test network script also provides the option to bring up the network using
 Certificate Authorities (CAs). In a production network, each organization would
 operate a CA (or multiple intermediate CAs) that creates the identities that
-belong to their organization. All of the identities created by a CA run by the
-organization would share the same root of trust. Although it takes more time to
-run the test network using CAs than to use cryptogen, bringing up a network using
-CAs can provide an introduction to a production network would be deployed.
-Standing up the Fabric CAs also provides you with the ability to enroll
-a client identity using the Fabric SDKs and create a certificate and private key
-that can be used by your application. Both cryptogen and the Fabric CAs generate
-the crypto material for each organization in the `organizations` folder.
+belong to their organization. All of the identities created by the CAs run by the
```

Comment:
In a production network, each organization 
operates a CA (or multiple intermediate CAs) that creates the identities that
belong to their organization. All of the identities created by a CA run by the organization share...

Reply:
done.